site/src/main/mdoc/overview.md

```diff
@@ -16,117 +16,116 @@ position: 10
 ## Session creation
 
 **neotypes** provides the `GraphDatabase` **factory** for creating a **Neo4j** `Driver`, which represents a connection with the Database.
-You can use this `Driver`, to create a `Session`, which can be used to perform operations _(`Transactions`)_ over the Database.<br>
-This **Scala** classes, are nothing more than simple wrappers over their **Java** counterparts. Which provide a more _"Scala-friendly"_ and functional API.
+You can use this `Driver` to perform operations _(`Transactions`)_ over the Database.<br>
+Those **Scala** classes, are nothing more than simple wrappers over their **Java** counterparts. But providing a more _"Scala-friendly"_ and functional API.
```

Comment:
Minor grammar things (but I'm not 100% confident 😁 )
1. the comma in `classes, are` should be removed. 
2. `. But providing ...` is part of the previous sentence